examples/federation/base-app/src/main/kotlin/com/expediagroup/graphql/sample/Application.kt

```diff
@@ -16,12 +16,30 @@
 
 package com.expediagroup.graphql.sample
 
+import com.expediagroup.graphql.federation.FederatedSchemaGeneratorConfig
+import com.expediagroup.graphql.federation.FederatedSchemaGeneratorHooks
+import com.expediagroup.graphql.federation.execution.FederatedTypeRegistry
+import com.expediagroup.graphql.sample.extension.CustomFederationSchemaGeneratorHooks
+import javax.validation.Validator
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
+import org.springframework.context.annotation.Bean
 
 @SpringBootApplication
-class Application
+class Application {
+    @Bean
+    fun federatedTypeRegistry() = FederatedTypeRegistry()
```

Comment:
this is the same as default so lets drop it -> https://github.com/ExpediaGroup/graphql-kotlin/blob/master/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/FederationAutoConfiguration.kt#L46